tgui/packages/tgui/interfaces/PipeDispenser.js

```diff
@@ -0,0 +1,102 @@
+import { useBackend, useLocalState } from '../backend';
+import { Button, Section, Flex, Box, NumberInput } from '../components';
+import { Window } from '../layouts';
+
+export const PipeDispenser = (props, context) => {
+  const { act, data } = useBackend(context);
+  const disposalpipes = data.disposalpipes || [];
+  const [amount, setAmount] = useLocalState(context, 'amount', 1);
+  const {
+    dispenser_ready,
+    windowName,
+    mobile,
+    removing_pipe,
+    laying_pipe,
+    max_disposal_pipes,
+  } = data;
+  return (
+    <Window
+      title={windowName}
+      width="300"
+      height={mobile ? 330 : 235}
+      fontFamily="Consolas"
+      font-size="10pt">
```

Comment:
While the props exist, broadly prefer not changing from defaults without a good reason. How does removing this affect the appearance?